rathemtn/Yilbit_Gerbigo.lua

```diff
@@ -0,0 +1,50 @@
+-- Yilbit Gerbigo
+-- Marr's Blessing
+-- items: 55035, 55036
+function event_say(e)
+	local fac = e.other:GetFaction(e.self);
+	
+	if(e.other:Race() ==  "Froglok" or fac < 5) then
+		if(e.message:findi("hail")) then
+			e.self:Say("Salutations. Come sit with me and have a drink. I am just going over some [" .. eq.say_link("business") .. "] right now.");
+		elseif(e.message:findi("business")) then
+			e.self:Say("You look like a stout follower of Marr's teachings. This document was just delivered to me. It seems the Paladins in Freeport wish for us to assist them. Let's see what this scroll says.");
+			e.self:Emote("begins to read the scroll.");
+			e.self:Say("Hmm, yes. Interesting. Most unfortunate. Have you ever heard of Castle [" .. eq.say_link("Mistmoore") .. "]?");
+		elseif(e.message:findi("mistmoore")) then
+			e.self:Say("It is an ancient castle on the continent of Faydwer, eternally covered in magical darkness. The castle's origins are shrouded in mystery to this day. According to this scroll, many years ago, Sir Bronthas, a Paladin of Marr took up the crusade to wipe Castle Mistmoore and its minions off the map once and for all. After a tiring journey he finally made his way to the castle. He was never heard from again. Rumors say that the brave if not foolish knight was killed and then raised from the dead to serve Lord Mistmoore. Still others say that his bones were used to decorate the dark halls of the castle. Truth be said, no one really knows what happened to him. That is why the Paladins in Freeport need our [" .. eq.say_link("assistance") .. "].");
+		elseif(e.message:findi("assistance")) then
+			e.self:Say("They wish for us to venture to the castle and find any sign of what happened to Sir Bronthas. I am afraid however that I was wounded in the war with the Trolls and can no longer fight and Shin Master Grubbus has his hands full training the influx of new recruits. Perhaps you could find some time to [" .. eq.say_link("look into this") .. "] for us?");
+		elseif(e.message:findi("look into this")) then
+			e.self:Say("You are a true follower of Marr. However I worry for your safety. If you do accept this task, please make sure you are ready. The castle is a deadly place. Should you find any trace of Sir Bronthas's remains, please bring them back to me. There is apparently a reward for completing this. Marr's blessing be with you.");
+		end
+	else
+		if(e.message:findi("hail")) then
```

Comment:
no need for the if hail check here, if not a Froglok or Faction any chat can just return the "Not something right" text